instances/technique/retinotopicMapping.jsonld

```diff
@@ -0,0 +1,14 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/technique/retinotopicMapping",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/Technique",
+  "definition": "'Retinotopic mapping' is a special type of receptive field mapping where the visual receptive field position is mapped in retinotopic coorinates in a brain region.",
```

Comment:
Better definition: "'Retinotopic mapping' is a stimulation technique in which neurons or neuron populations within the visual stream are mapped point-to-point to visual input of the retinal surface.

@tgbugs also here I think we need you feedback for a nice definition.